### PR TITLE
sparkline is added, last version is ready

### PR DESCRIPTION
**Pull request by alisaitbilgi**
In this update, sparkline is added and all requirements tried to be satisfied.

### CODE REVIEW

**Review by alisaitbilgi — COMMENTED** [review]
It's ready to be reviewed

File: public/styles/style.css [hunk 1]
```diff
@@ -140,11 +143,107 @@ img {
...
 }
 
+.centered {
+    margin: 0 auto;
+    padding-top: 20%;
+    font-size: 20px;
+}
+
+
+@media (max-width: 850px) {
```
Comment (bulentzeki) [inline public/styles/style.css]: why 850px? There are standards for media queries.

File: public/styles/style.css [hunk 2]
```diff
@@ -140,11 +143,107 @@ img {
...
+    .main > nav {
```
Comment (bulentzeki) [inline public/styles/style.css]: Please learn what `>` does if you don't know.

File: src/components/HomePresenter.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+export default class ContentContainer extends Component {
+  render() {
+    let path;
+    if (this.props.takeUserName) {
+      path = `/users/${this.props.takeUserName}`;
+    } else {
+      path = "/users";
+    }
+    return (
+        <iframe src={path} className="content" scrolling="no" />
```
Comment (bulentzeki) [inline src/components/HomePresenter.js]: Just a suggestion
```
<iframe className="content" 
             src={this.props.takeUserName ? `/users/${this.props.takeUserName}` : "/users"} 
            scrolling="no" />
```

File: src/reducers/singleReducer.js [hunk 4]
```diff
@@ -1,9 +1,11 @@
 import I from "immutable";
 
-export default function singleReducer(state = I.fromJS({responseObject: I.List()}), action) {
+export default function singleReducer(state = I.fromJS({username: "", responseObject: I.List()}), action) {
```
Comment (bulentzeki) [inline src/reducers/singleReducer.js]: Avoid initializations in reducer

**Review by bulentzeki — CHANGES_REQUESTED** [review]

**Review by bulentzeki — APPROVED** [review]

**Pull request merged**